### ISSUE TITLE
when changing from viewing my observations to all observations, icon change

### ISSUE THREAD

**Issue by bjornhjelle** [Biocaching/app#39]
when changing from viewing my observations to all observations, two icons on the row below the top row disappear.

**Comment by pbb72** [comment]
That is correct, those are the friends and the likes pages. Maybe it's confusing to combine these with the personal observation page?
